spec/spec_helper.rb

```diff
@@ -18,7 +18,3 @@ def input(plugin, &block)
   end
 
 end
-
-RSpec.configure do |c|
-  c.include IrcHelpers
```

Comment:
my bad there, I caused this problems. what we should improve from it, this
means because of the way we run the plugins integration test (all at once)
we should still be able to add helpers for plugins (specific ones) that
could not be living in devutils. For now I agree your fix makes sense, and
we can just update other plugins to looks alike.

- purbon

On Wed, Oct 7, 2015 at 4:26 PM João Duarte <notifications@github.com> wrote:

> In spec/spec_helper.rb
> <https://github.com/logstash-plugins/logstash-input-irc/pull/17#discussion_r41396339>
> :
>
> > @@ -18,7 +18,3 @@ def input(plugin, &block)
> >    end
> >
> >  end
> > -
> > -RSpec.configure do |c|
> > -  c.include IrcHelpers
>
> yep, we have the same problem in ganglia, udp, gelf, etc, specially the
> ones that redefine the input helper method to have 2 arguments + block
> instead of 1 argument + block
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/logstash-plugins/logstash-input-irc/pull/17/files#r41396339>
> .
>